cmake/EthCompilerSettings.cmake

```diff
@@ -22,6 +22,13 @@ if(CCACHE_FOUND)
 	message("Using ccache")
 endif(CCACHE_FOUND)
 
+include(CheckCXXCompilerFlag)
+
+check_cxx_compiler_flag(-fstack-protector-strong stack_protector_support)
+if (stack_protector_support)
+	add_compile_options(-fstack-protector-strong)
```

Comment:
This new way doesn't pass the `-fstack-protector` option anymore as it was the case before. According to gcc it seems that `-strong` implies the basic one too. Is that correct?